### ISSUE TITLE
Feature request: Automatically play next track when removing track from playlist

### ISSUE THREAD

**Issue by pcwilcox** [vdesabou/alfred-spotify-mini-player#522]
If I remove a track from a playlist, I don't want to keep listening to it. I wait long enough to confirm that the mini player removed it, then I hit next. It would be very cool if the player did this for me.

**Comment by vdesabou** [comment]
That makes sense.

Just to clarify, it's only when you remove the current track from a playlist, right ?

![CleanShot 2021-09-03 at 09 05 58](https://user-images.githubusercontent.com/4061923/131964641-db20e304-e3ea-4cf8-b3f8-00dac9a0dc25.jpg)

**Comment by pcwilcox** [comment]
Yes, that's right. I'm not sure about others but the flow I use is to add far too many tracks to a single playlist (many thousands of them) and then listen to it on shuffle and remove any that aren't appealing for whatever reason.

**Comment by vdesabou** [comment]
I'll add this in next release `10.18`

**Comment by gimbo** [comment]
So, I've just discovered that this doesn't work if I use the command to "Remove Current Track from Alfred Playlist or Your Music" — it only seems to work if I use "Remove current track from..." and select the playlist by hand.

I'm triggering the former with a hotkey (targetting my Alfred Playlist), so I don't get any benefit from this fix. I wonder if it could be extended to the Alfred Playlist too?

**Comment by vdesabou** [comment]
@gimbo I've created #535 to track this

**Comment by amarukot** [comment]
this new feature is great because I've thought about this before. But I'm hoping to make it optional or at least as a separate action? Sometimes you just want to remove from playlist but DO want to keep listening. 

remove_current_track_from_and_play_next
remove_current_track_from

e.g. I'm listening to a "new songs" playlist and I hear a song that I want to move to a different playlist. I want to remove from "new songs" playlist but want to keep listening. Is this possible?
